prototypes/prototype.adoc

```diff
@@ -0,0 +1,44 @@
+---
+authors: Han Solo <han@example.com>, Frodo Baggins <frodo@example.com>
+state: predraft
+---
+= RFD <NUMBER> <Title>
+:author: Han Solo <han@example.com>; Frodo Baggins <frodo@example.com>
+:state: predraft
+:revremark: State: {state}
+:attribute-missing: warn
```

Comment:
Can we have a justification for any/all of these attributes? It would be nice if we could have *none* of them to have let distraction from the content when reading/editing the raw files.